x-pack/plugin/esql/qa/testFixtures/src/main/resources/ignore-case.csv-spec

```diff
@@ -76,3 +76,36 @@ from employees | where first_name =~ "Georg?" | sort emp_no | keep emp_no, first
 
 emp_no:integer | first_name:keyword
 ;
+
+
+fieldRight#[skip:-8.12.99, reason:case insensitive operators implemented in v 8.13]
+from employees | where "Guoxiang" =~ first_name | keep emp_no, first_name;
+
+emp_no:integer  | first_name:keyword
+10015           | Guoxiang
+;
+
+
+expressionsRight#[skip:-8.12.99, reason:case insensitive operators implemented in v 8.13]
+from employees | where first_name =~ concat("Tzv","ETAN") | keep emp_no, first_name;
+
+emp_no:integer  | first_name:keyword
+10007           | Tzvetan
+;
+
+
+expressionsLeft#[skip:-8.12.99, reason:case insensitive operators implemented in v 8.13]
+from employees | where concat(first_name, "_foo") =~ "TzvETAN_fOo" | keep emp_no, first_name;
+
+emp_no:integer  | first_name:keyword
+10007           | Tzvetan
+;
+
+
+expressionsLeftRight#[skip:-8.12.99, reason:case insensitive operators implemented in v 8.13]
+from employees | where substring(first_name, 1, 2) =~ substring(last_name, -2) | keep emp_no, first_name, last_name | sort emp_no;
```

Comment:
I've thought about this some more and we need to conclude how to address the missing requirements (multi-value and wildcard matching) from
https://github.com/elastic/elasticsearch/issues/103599 *before* enabling this functionality.
If `=~` is asymmetric like EQL `:` than comparing two fields can't happen. in ESQL we might do thing differently but until we reach a conclusion, I prefer we don't ship these features to begin with; it's better to enable them later than have them disabled.

In other words, please keep the comparison only between a literal/folded expression and a field but not between two fields.

Reply:
> in ESQL we might do thing differently but until we reach a conclusion, I prefer we don't ship these features to begin with

👍 I think it's a wise decision, I'll disable field vs field comparison at validation time (I'll allow only foldable expressions on the right) until we have a final decision on all the aspects.